lib/src/calendar_event_data.dart

```diff
@@ -96,4 +96,156 @@ class CalendarEventData<T extends Object?> {
 
   @override
   int get hashCode => super.hashCode;
+
+  /// Checks if the eventData is the same 
+  /// without checking startTime and endTime.
+  ///
+  /// This is the workaround because [CalendarEventData] does not 
+  /// have a unique ID.
+  /// 
+  /// - It can create a bug when 2 events have are exactly the same except 
+  ///   for the startTime and endTime.
+  ///
+  bool compareWithoutTime(Object other) {
```

Comment:
So this is the best way I could come up with to find the CalendarEventData when the start/end time keeps changing.